put_get_test.go

```diff
@@ -642,6 +642,7 @@ func TestPutLargeFile(t *testing.T) {
 
 func TestPutGetMaxLOBSize(t *testing.T) {
 	// the LOB sizes to be tested
+	// set "alter session set ALLOW_LARGE_LOBS_IN_EXTERNAL_SCAN = true", when testing for increased max LOB size
```

Comment:
Instead of this comment maybe you can comment `dbt.mustExec`?

Reply:
ok, changed the comment.

Reply:
I have one more thought - shouldn't we test with this param set to true? I believe you don't want to enable it by default, because on may envs we don't have permissions to set it? So maybe we can do something like:
```
_, err := dbt.Exec("alter session set ALLOW_LARGE_LOBS_IN_EXTERNAL_SCAN = true")
if err != nil {
   // log err
  // t.Skip
}
```